vault/login_mfa.go

```diff
@@ -1361,6 +1361,10 @@ func (b *LoginMFABackend) mfaLoginEnforcementConfigByNameAndNamespace(name, name
 func (b *LoginMFABackend) mfaLoginEnforcementConfigToMap(eConfig *mfa.MFAEnforcementConfig) (map[string]interface{}, error) {
 	resp := make(map[string]interface{})
 	resp["name"] = eConfig.Name
+	ns, err := b.namespacer.NamespaceByID(context.Background(), eConfig.NamespaceID)
```

Comment:
Should we handle the error if it is not nil?

Reply:
I'm not really sure - I don't like failing silently but similarly, the endpoint wouldn't have complained before and would have e.g. returned error, so I thought it best to keep the previous behaviour somewhat consistent. I'm happy to change to e.g. returning the error if you feel it's appropriate!

Reply:
Yeah, this check is done all over the place. My preference would be to do it here as well. But, I don't have a strong feeling about it.

Reply:
Updated!

Reply:
How do you feel about changing this logic to the following instead?

```
ns, err := b.namespacer.NamespaceByID(context.Background(), eConfig.NamespaceID)
if ns == nil || err != nil {
 return nil, err
}

resp["namespace_path"] = ns.Path
...
```

It's a bit more concise, but the same logic, so feel free to ignore :).

Reply:
Good suggestion, changed!